proto/src/main/proto/grpc.proto

```diff
@@ -504,7 +504,7 @@ message CancelOfferReply {
 message OfferInfo {
     string id = 1;
     string direction = 2;
-    uint64 price = 3;
+    double price = 3;
```

Comment:
`price` and `trigger_price` should be handled consistently.

We should actually build this PR on the [latest rebase](https://github.com/haveno-dex/haveno/pull/312) so it has the latest upstream changes and avoids merge conflicts. Note these price fields change to strings in the rebase.

I have not checked, but a primary goal of this PR is that the base and counter currencies are always stored correctly in the models, e.g. OfferPayload, so confirming this is the case. Ideally then we fix all inverted instances within the application.

Reply:
OK, I'll rebase and give it a shot, it will take awhile to verify that it works due to the inversions in scenarios such as price triggering and the Volume calculation classes needs changes as well.

Reply:
To clarify, do we want the price inversion to also disappear in the internal daemon models? This is the much more challenging part of the task. The base and counter currencies can be un-inverted pretty easily.